Please add support for sly==0.5.0

Please see our "Development Speed" note in the README.md of the project for expected response times. 

Thanks for the feature request! Here's some guidance to get you started on what we are looking for in terms of details:

**Is your feature request related to a problem? Please describe.**
We are a Linux distro and can only ever ship one version of a python package.

We recently updated sly from 0.4 to 0.5 and `test_command_line` has started breaking.

```
scim2-filter-parser> Executing unittestCheckPhase
scim2-filter-parser> ............................F..........................................................................................................................................
scim2-filter-parser> ======================================================================
scim2-filter-parser> FAIL: test_command_line (tests.test_lexer.CommandLine)
scim2-filter-parser> ----------------------------------------------------------------------
scim2-filter-parser> Traceback (most recent call last):
scim2-filter-parser>   File "/build/source/tests/test_lexer.py", line 449, in test_command_line
scim2-filter-parser>     self.assertEqual(result, expected)
scim2-filter-parser> AssertionError: Lists differ: ["Tok[50 chars]dex=0, end=8)", "Token(type='EQ', value='eq', [96 chars]21)"] != ["Tok[50 chars]dex=0)", "Token(type='EQ', value='eq', lineno=[73 chars]12)"]
scim2-filter-parser> 
scim2-filter-parser> First differing element 0:
scim2-filter-parser> "Token(type='ATTRNAME', value='userName', lineno=1, index=0, end=8)"
scim2-filter-parser> "Token(type='ATTRNAME', value='userName', lineno=1, index=0)"
scim2-filter-parser> 
scim2-filter-parser> - ["Token(type='ATTRNAME', value='userName', lineno=1, index=0, end=8)",
scim2-filter-parser> ?                                                             -------
scim2-filter-parser> 
scim2-filter-parser> + ["Token(type='ATTRNAME', value='userName', lineno=1, index=0)",
scim2-filter-parser> -  "Token(type='EQ', value='eq', lineno=1, index=9, end=11)",
scim2-filter-parser> ?                                                 --------
scim2-filter-parser> 
scim2-filter-parser> +  "Token(type='EQ', value='eq', lineno=1, index=9)",
scim2-filter-parser> -  "Token(type='COMP_VALUE', value='bjensen', lineno=1, index=12, end=21)"]
scim2-filter-parser> ?                                                               --------
scim2-filter-parser> 
scim2-filter-parser> +  "Token(type='COMP_VALUE', value='bjensen', lineno=1, index=12)"]
scim2-filter-parser> 
scim2-filter-parser> ----------------------------------------------------------------------
scim2-filter-parser> Ran 167 tests in 0.104s
scim2-filter-parser> 
scim2-filter-parser> FAILED (failures=1)
```

**Describe the solution you'd like**
Make the package compatible with `sly==0.5`

**Describe alternatives you've considered**
Unfortunately we can only disable the package or leave it in a broken state.

**Additional context**
n/a

cc @s1341

--- a/.github/workflows/cicd.yml
+++ b/.github/workflows/cicd.yml
@@ -33,7 +33,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - run: pip install tox poetry
+      - run: pip install -U tox poetry
       - run: tox -e flake8
 
   coverage:

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "scim2-filter-parser"
-version = "0.4.2"
+version = "0.5.0"
 description = "A customizable parser/transpiler for SCIM2.0 filters."
 license = "MIT"
 authors = ["Paul Logston <paul@15five.com>"]
@@ -36,7 +36,7 @@ sfp-query = 'scim2_filter_parser.queries.sql:main'
 
 [tool.poetry.dependencies]
 python = ">=3.8"
-sly = "==0.4"
+sly = "==0.5"
 
 django = { version = ">=3.2", optional = true }
 

--- a/tests/test_lexer.py
+++ b/tests/test_lexer.py
@@ -442,9 +442,9 @@ class CommandLine(TestCase):
         lexer.main(['userName eq "bjensen"'])
         result = self.test_stdout.getvalue().strip().split('\n')
         expected = [
-            "Token(type='ATTRNAME', value='userName', lineno=1, index=0)",
-            "Token(type='EQ', value='eq', lineno=1, index=9)",
-            "Token(type='COMP_VALUE', value='bjensen', lineno=1, index=12)",
+            "Token(type='ATTRNAME', value='userName', lineno=1, index=0, end=8)",
+            "Token(type='EQ', value='eq', lineno=1, index=9, end=11)",
+            "Token(type='COMP_VALUE', value='bjensen', lineno=1, index=12, end=21)",
         ]
         self.assertEqual(result, expected)
 

